2020-02-Spring/2_linked_lists/README.md

```diff
@@ -126,31 +147,117 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
-
-Source: TODO :bug:
+### 2. Remove Zero Sum Consecutive Nodes from Linked List
 
 #### Naive/Simple Solution
+The idea is to locate the sublist of nodes that sum to 0 and de-list them.<br>
```

Comment:
I personally found this explanation hard to follow. Please consider revising it

Reply:
I second this. The mechanics of deleting nodes and resetting list pointers appropriately seem fine, but your explanation glosses over how you actually find the zero-sum sublist. It is _this_ component which we want to hear about, not necessarily Linked List manipulation mechanics.